config/initializers/webpacker.rb

```diff
@@ -0,0 +1,6 @@
+# Memory needed presently to compile js/ts is aroung 2.5GB
```

Comment:
Thing about this is that `--max-old-space-size` is only a temporary workaround for an old version of node. Do we still want to make an initializer for it? Memory limit has been a very well-known problem with older nodejs but AFAIK it's kinda fixed in the present. At least according to [this article](https://medium.com/geekculture/node-js-default-memory-settings-3c0fe8a9ba1), the default limit in `>= 14.x` seems to be `4.0 GB`.
I haven't found any mention of it but in node 12 everybody is basically complaining about the hard-coded limit of 2GB so I would be surprised if more recent versions kept this limit (which is confirmed by the article not to be the case).

Copying table for reference:
```
Node.js Version   Limit  
----------------- -------
  18.x             4.0 GB
  17.x             4.0 GB
  16.x             4.0 GB
  15.x             4.0 GB  
  14.x             4.0 GB  
  13.x             2.0 GB  
  12.x             2.0 GB  
  11.x             1.4 GB  
  10.x             1.4 GB  
  9.x              1.4 GB
  ```

Reply:
We can remove this once we upgrade. But also keeping the setting may help us identify big regressions in memory usage that we have.

I think that at least it is worth to make things work automatically without extra variables on command line, as it is very frustrating to see things fail without proper error.